### ISSUE TITLE
javascript.builtins.Intl.NumberFormat.NumberFormat - Missing NumberFormat v3 tables

### ISSUE THREAD

**Issue by aloisklink** [mdn/browser-compat-data#15213]
#### What information was incorrect, unhelpful, or incomplete?

It looks like the features of `Intl.NumberFormat` v3 (added in https://github.com/mdn/content/pull/10783) are missing from the compatibility tables.

#### What did you expect to see?

Compatibility table entries for new parameters such as `roundingMode`.

#### Did you test this? If so, how?

According to the docs, the following line should use `trunc` rounding (truncation) to `9.99`, however on Node v16 and Chromium: 98.0.4758.102, this instead rounds to `10`.

```js
new Intl.NumberFormat('en-US', {roundingMode: "trunc", maximumFractionDigits: 2}).format(9.999)
```

Looking at https://chromestatus.com/feature/5707621009981440#details, it doesn't look like any other browser has this feature enabled either.

<details>
<summary>MDN page report details</summary>

* Query: `javascript.builtins.Intl.NumberFormat.NumberFormat`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat
* Report started: 2022-03-01T11:30:18.190Z

</details>

**Comment by Josh-Cena** [comment]
Fixed by #16840

**Comment by aloisklink** [comment]
> Fixed by https://github.com/mdn/browser-compat-data/pull/16840

Agreed! Looks like they've been added! Thanks for the update!
